### ISSUE TITLE
(SECURITY) Your client's secret token for VK is visible!

### ISSUE THREAD

**Issue by sasdallas** [IgorTimofeev/MineOS#516]
Hello, @IgorTimofeev.

I was fooling around on OpenComputers with MineOS (very good job btw, love the OS), and discovered your VK secret is visible.
If you open the VK app and type in random garbage into email and password, it returns a HTTP response code 401, then prints the URL.

Problem is: the URL contains your client_secret and client_id as parameters.
Maybe parse an HTTP response code 401 as a failure to login for VK?

Attached image:

![image](https://user-images.githubusercontent.com/40303843/208268467-52df79e7-09eb-4f18-a05d-5435f0d8a722.png)

**Comment by CoolCat467** [comment]
Well yes, this is how you are able to log in to VK. This is following everything that the [official VK doccumentation](https://dev.vk.com/api/getting-started) says to. According to people on StackOverflow [here](https://stackoverflow.com/questions/499591/are-https-urls-encrypted), everything in the URL is encrypted. The only thing anyone watching the network can see is the domain name you are attempting to connect to, which in this case is `oauth.vk.com`.

Basically, there is nothing security wise out of the ordinary here and this is not an issue.

**Comment by IgorTimofeev** [comment]
> your VK secret is visible

It's not mine, it was taken from decompiling official android app, so.. :p

> Maybe parse an HTTP response code 401 as a failure to login for VK?

Ok, one moment, I'll delete URL showing from login UI
